mods/pageviews.yaml

```diff
@@ -0,0 +1,27 @@
+paths:
+    /per-article/{project}/{access}/{agent}/{article}/{granularity}/{start}/{end}:
+        get:
+          summary: query pageviews per article
+          operationId: pageviewsForArticle
+    /per-project/{project}/{access}/{agent}/{granularity}/{start}/{end}:
+        get:
+          summary: query pageviews per project
+          operationId: pageviewsForProjects
+    /top/{project}/{access}/{year}/{month}/{day}:
+        get:
+          summary: query top pageviews
+          operationId: pageviewsForTops
+
+    # These three endpoints are just for fake data:
```

Comment:
All of these endpoints could be removed from the module and be completely spec-driven in the `analytics_test` config ([See docs](https://github.com/wikimedia/restbase/blob/master/doc/HandlerConfiguration.md))